internal/interpreter/highlight.rs

```diff
@@ -315,10 +315,7 @@ fn find_element_at_offset(component: &Rc<Component>, path: PathBuf, offset: u32)
 fn repeater_path(elem: &ElementRc) -> Option<Vec<String>> {
     let enclosing = elem.borrow().enclosing_component.upgrade().unwrap();
     if let Some(parent) = enclosing.parent_element.upgrade() {
-        if parent.borrow().repeated.is_none() {
-            // This is not a repeater, it might be a popup menu which is not supported ATM
-            return None;
-        }
+        parent.borrow().repeated.as_ref()?;
```

Comment:
I don't understand clippy rule. Clippy doesn't like when I'm using map() on option without using the result,  but it's ok to use as_ref like that without using the result.
Anyway... Ok even if I don't like it.

But the comment should stay. Need to be adjusted it is about bailing out.